spec/yaml/swiftnav/sbp/acquisition.yaml

```diff
@@ -70,3 +70,50 @@ definitions:
             desc: |
               PRN-1 identifier of the satellite signal for which
               acquisition was attempted
+
+ - MSG_ACQ_SV_PROFILE:
+    id: 0x001E
+    short_desc: Reacq perfomance measurement and debug
+    desc: |
+      The message describes SV profile during acquisition time.
+      The message is useful to debug and measure the performance of
+      the new (re-)acq design.
+    public: true
+    fields:
+        - N:
```

Comment:
The motivation behind sending the acq results for multiple SVs in one SBP was to reduce the overhead related to SBP header and footer, which is 8 bytes in total. One SV acq data is 21 bytes. So the overhead is 8/23 -> 38%. Combining multiple acq data SVs into one SBP can reduce it to 8/255 -> ~3%. We expect quite a large number of these messages to be send per second, when there are few tracked SVs.